src/lib/viewers/media/DashViewer.js

```diff
@@ -152,10 +153,11 @@ class DashViewer extends VideoBaseViewer {
         this.adapting = true;
         this.player = new shaka.Player(this.mediaEl);
         this.player.addEventListener('adaptation', this.adaptationHandler);
+        this.player.addEventListener('streaming', this.shakaManifestHandler);
         this.player.addEventListener('error', this.shakaErrorHandler);
         this.player.configure({
             abr: {
-                enabled: true
+                enabled: false
```

Comment:
Setting this to false by default, it will get overridden if 'auto' is saved in our cache as the current quality setting.